website/addons/gitlab/serializer.py

```diff
@@ -0,0 +1,60 @@
+from website.addons.base.serializer import StorageAddonSerializer
+
+from website.util import api_url_for
+
+from website.addons.gitlab.api import GitLabClient
+from website.addons.gitlab.exceptions import GitLabError
+
+from framework.sessions import session
+
+class GitLabSerializer(StorageAddonSerializer):
+
+    addon_short_name = 'gitlab'
+
+    # Include host information with more informative labels / formatting
+    def serialize_account(self, external_account):
+        ret = super(GitLabSerializer, self).serialize_account(external_account)
+        key = external_account.oauth_key
+        secret = external_account.oauth_secret
+        host = external_account.display_name
+        session.data['oauth_client_id'] = key
```

Comment:
This additional session data:
1) Doesn't appear to be saved, preventing it from being used anywhere else
2) Doesn't appear to be used anywhere else, with the possible exception of `oauth_host` in one place
3) Probably shouldn't be stored on the user's session.